src/apps/dictionaries/components/__test__/ReleasedVersions.test.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import ReleasedVersions from '../../../../apps/dictionaries/components/ReleasedVersions';
+import {render} from '@testing-library/react';
+import '@testing-library/jest-dom'
+import {
+    BrowserRouter as Router,
+} from "react-router-dom";
+
+type releasedVersionProps = React.ComponentProps<typeof ReleasedVersions>;
+
+const baseProps: releasedVersionProps = {
+    versions: [],
+    subscriptionUrl: "Subscription URL",
```

Comment:
subscriptionUrl and linkedSource are removed from interface Props defined for ReleasedVersions.
We should remove these from the test cases

Reply:
Done @kirity-mahanty